tests/test_pe_reports.py

```diff
@@ -21,11 +22,24 @@
     "critical",
 )
 
+# TODO: Setup log rotate to rotate pe-reports log file
+#  Issue - https://github.com/cisagov/pe-reports/issues/248
+
+# Setup logging to file
+logging.basicConfig(
+    filename=CENTRAL_LOGGING_FILE,
+    filemode="a",
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S",
+    level="info",
```

Comment:
This should be consistent with the other tests:
```suggestion
    level="INFO",
```